packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -41,7 +41,7 @@ function render(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchF
 
 function renderReverse(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
   return (
-    <SearchWithin label="Search" {...props}>
+    <SearchWithin label="Search within" {...props}>
```

Comment:
Nit:
Sorry about these iffy labels from my test branch, perhaps change them to "Test label" or something easily distinguishable/realistic
```suggestion
    <SearchWithin label="Search within" {...props}>
```